package.json

```diff
@@ -11,21 +11,22 @@
     "test": "jest",
     "perf": "npx tsx tests/perf/*.ts",
     "test:coverage": "jest --coverage",
-    "format": "prettier --ignore-unknown --write .",
+    "format": "biome format --write .",
     "lint": "eslint",
     "lint:fix": "eslint --fix",
     "prepare": "husky"
   },
   "lint-staged": {
     "**/*": [
       "eslint --fix",
-      "prettier --ignore-unknown --write"
+      "biome check --fix"
```

Comment:
Do we not want only [staged files](https://biomejs.dev/guides/integrate-in-vcs/#process-only-staged-files)?